### ISSUE TITLE
HTTP error when using turbopack and Next.js 13

### ISSUE THREAD

**Issue by usmanity** [nextauthjs/next-auth#6327]
### Environment

System:
    OS: macOS 13.0.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 1.09 GB / 32.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.12.1 - ~/Library/Caches/fnm_multishells/89629_1664427879554/bin/node
    Yarn: 1.22.19 - ~/Library/Caches/fnm_multishells/89629_1664427879554/bin/yarn
    npm: 8.19.2 - ~/Library/Caches/fnm_multishells/89629_1664427879554/bin/npm
  Browsers:
    Brave Browser: 108.1.46.144
    Chrome: 108.0.5359.124
    Firefox: 108.0.1
    Safari: 16.1
  npmPackages:
    next: 13.1.1 => 13.1.1
    next-auth: ^4.18.7 => 4.18.7
    react: 18.2.0 => 18.2.0

### Reproduction URL

private-setup-see-below

### Describe the issue

I have a barebones app set up with a login page that uses the `signIn` function with an email (magic link) provider.

The setup works fine when running `next dev` without `--turbo` but when I add the `--turbo` flag, I see the following error: 

```
Error: This action with HTTP GET is not supported by NextAuth.js
```

The only difference is when I run the app without `--turbo`, I am fine making requests to the `signIn` route. I'm not sure what other info would be helpful as I'm super new to Next.js 13 but I wanted to report it in case the NextAuth team has any insights or info on this. 

### How to reproduce

- Create a next.js 13 app
- Add NextAuth with a login page and email provider
- Try submitting an email address

### Expected behavior

Should work as it does without turbopack.

**Comment by DenisBessa** [comment]
I'm having the exact same issue here

**Comment by Sushant5776** [comment]
Same here!

**Comment by DenisBessa** [comment]
The problem seems to have been resolved with version 13.2 of Next.js

@Sushant5776 @usmanity

**Comment by godfrednanaowusu** [comment]
The issue is still there even with Nextjs 13.3
